modules/custom/az_digital_asset_library/config/quickstart/ib_dam.settings.yml

```
@@ -0,0 +1,6 @@
+dependencies:
+  enforced:
+    module:
+      - az_digital_asset_library
+debug: false
+staging: true
```

```suggestion
staging: false
```
According to Garrett, the staging environment should no longer be required.